src/change_stream.ts

```diff
@@ -368,7 +346,10 @@ export class ChangeStream extends EventEmitter {
     if (!this.cursor) {
       throw new MongoError('ChangeStream has no cursor, unable to pause');
     }
-    return this.cursor.pause();
+    const stream = this.cursor[kCursorStream];
```

Comment:
`pause` and `resume` are methods providing API parity with Node streams, they should be removed. If users want to use the change stream as a stream, they should call the `stream` method.